include/tester/ssp/Softphone.hpp

```diff
@@ -66,7 +65,8 @@ class Softphone
 
 private:
     static constexpr auto SIP = "sip:";
-
+    std::function<void(pj::CallInfo ci, const pj::OnCallStateParam &prm)> boundOnCallState =
+        std::bind(&Softphone::onCallState, this, std::placeholders::_1, std::placeholders::_2);
     SSPAccount _account;
     SSPCall *_call;
```

Comment:
Now for the next challenge: make this member of type `std::unique_ptr<SSPCall>` rather of type `SSPCall*` - our team standard says to minimize C-style writing and to mainly use the C++ standard library.

so read about `std::unique_ptr` and what the function `std::make_unique` does.

Reply:
done